base_tier_validation/models/tier_validation.py

```diff
@@ -576,13 +581,13 @@ def _notify_accepted_reviews_body(self):
         )
         if has_comment:
             comment = has_comment.mapped("comment")[0]
-            return _("A review was accepted. (%s)") % comment
-        return _("A review was accepted")
+            return self.env._("A review was accepted. (%(comment)s)", comment=comment)
```

Comment:
Same as previous comments. It's a shame to invalidate existing translatable terms just to use a kword argument on a term with only 1 arg